docs/install-manual.md

```diff
@@ -10,7 +10,7 @@ This section tells you how to create a Docker Swarm on your network to use only
 You install Docker Swarm on a single system which is known as your Docker Swarm manager. You create the cluster, or swarm, on one or more additional nodes on your network. Each node in your swarm must:
 
 * be accessible by the swarm manager across your network
-* a TCP port open to listen for the swarm manager 
+* Docker Engine 1.6.0+ installed listenning on an open TCP port for the swarm manager
```

Comment:
Hi Vieux, Spelling error.  I'd like to split the second bullet into two.  Each point starts with a verb so:

* be accessible by the swarm manager across your network
* have Docker Engine 1.6.0+ installed
* open a TCP port to listen for the manager